src/content/docs/en/release/diff-fedora.mdx

```diff
@@ -15,6 +15,11 @@ Changes listed below are nonexhaustive and cumulative, meaning each section cont
 - 🛳️ This ships existing Fedora packages
 - 🔥 This is a significant breaking change
 
+## Ultramarine 41
+
+- 🔥 Switched to Readymade from Anaconda
+- 🔥 Switched to Raspberry Pi Foundation Kernel
```

Comment:
Add Pi/raw ARM images will be BTRFS

Can also include Chromebook change, will be worth looking at closer to release.

Reply:
add those

Reply:
Will later